tests/resources/bergamo/expected_session.json

```diff
@@ -2079,7 +2301,2019 @@
         "excitation_power": "20",
         "excitation_power_unit": "percent"
       },
-      "output_parameters": {},
+      "output_parameters": {
+        "tiff_files": [
```

Comment:
This tiff_files is 2000 lines long, is that expected?

Reply:
yes it is! :)